bin/postgres-ha/pre-bootstrap.sh

```diff
@@ -220,17 +220,26 @@ set_pg_user_credentials() {
         env_check_err "PATRONI_SUPERUSER_PASSWORD"
     fi
 
-    if [[ -d "/pgconf/pguser" ]]
+    if [[ -d "/pgconf/pgreplicator" ]]
     then
         echo_info "Setting 'replicator' credentials using file system"
         
+        # Configure certificate-based authentication for replication if proper certs are available.
+        # Otherwise use a password
+        if [[ -f "/pgconf/replicator.key" ]] && [[ -f "/pgconf/replicator.crt" ]] && [[ -f "/pgconf/ca.crt" ]]
+        then
+            export PATRONI_REPLICATION_SSLKEY="${PATRONI_POSTGRESQL_DATA_DIR}/replicator.key"
+            export PATRONI_REPLICATION_SSLCERT="${PATRONI_POSTGRESQL_DATA_DIR}/replicator.crt"
+            export PATRONI_REPLICATION_SSLROOTCERT="${PATRONI_POSTGRESQL_DATA_DIR}/ca.crt"
```

Comment:
This block is missing a reference to the `ca.crl` file. This is optional, but we should include it in one of the variables if it exists.

Reply:
The CRL file is now included.